pkg/cmd/create_social_event.go

```diff
@@ -101,13 +112,13 @@ func CreateSocialEvent(ctx *clicontext.CommandContext, startDate, endDate, descr
 			Name:      code,
 		},
 		Spec: toolchainv1alpha1.SocialEventSpec{
-			StartTime:         metav1.NewTime(start),
-			EndTime:           metav1.NewTime(end),
-			MaxAttendees:      maxAttendees,
-			UserTier:          userTier,
-			SpaceTier:         spaceTier,
-			Description:       description,
-			PreferSameCluster: preferSameCluster,
+			StartTime:     metav1.NewTime(start),
+			EndTime:       metav1.NewTime(end),
+			MaxAttendees:  maxAttendees,
+			UserTier:      userTier,
+			SpaceTier:     spaceTier,
+			Description:   description,
+			TargetCluster: targetCluster,
```

Comment:
this is a name used in ksctl.yaml file - it's a short-cut that we use in command-line, but it doesn't have any meaning in the cluster. 

We need to use the name of the ToolchainCluster CR instead. You can use either the current approach that is used in the `retarget` command https://github.com/kubesaw/ksctl/blob/2cf7e16c94f0227f58cba32d33f3e4fa98cc0fe2/pkg/cmd/retarget.go#L104-L106 or get the ToolchainCluster CR and get the name from it.

Reply:
Good catch. Addressed here: https://github.com/kubesaw/ksctl/pull/59/commits/89b7a0fbd944ec862e33646db5b12739dbae30dc